code/src/main/resources/main.es6

```diff
@@ -1,7 +1,10 @@
 const libs = {
   repository: require("/lib/repository"),
   access: require("/lib/access"),
-  util: require("/lib/util")
+  util: require("/lib/util"),
+  eventLib: require("/lib/xp/event"),
```

Comment:
Small nitpick, but perhaps rename "eventLib" to "event" to keep the naming consistent with the others there? :)

Reply:
Nitpick fixed!